src/trials/showEarnings.js

```diff
@@ -0,0 +1,51 @@
+import { pdSpotEncode, photodiodeGhostBox } from "../lib/markup/photodiode"
+import { addCursor } from "../lib/utils"
+import { envConfig } from "../config/main"
+import { jsPsych } from "jspsych-react/dist/experiment"
+import { eventCodes } from "../config/trigger"
+const { earningsDisplay } = require("../lib/markup/earnings")
+
+/**
+ * Displays the earnings from a trial.
+ * @param {number} duration How long to display the earnings.
+ * @returns trial The jsPsych trial object.
+ */
+const showEarnings = (duration) => {
+  // Maps colors to their corresponding key code.
+  const colorKeyMappings = {
+    red: 82,
```

Comment:
should the key map be part of the config instead? if someone added `'green'` to the config, would it work?

Reply:
Actually fixed this in the next commit!